subsys/net/ip/route.c

```diff
@@ -668,16 +706,24 @@ int net_route_mcast_foreach(net_route_mcast_cb_t cb,
 }
 
 struct net_route_entry_mcast *net_route_mcast_add(struct net_if *iface,
-						  struct in6_addr *group)
+						  struct in6_addr *group,
+						  uint8_t prefix_len)
 {
 	int i;
 
+	if ((!net_if_flag_is_set(iface, NET_IF_FORWARD_MULTICASTS)) ||
+			(!net_ipv6_is_addr_mcast(group)) ||
+			(net_ipv6_is_addr_mcast_iface(group)) ||
+			(net_ipv6_is_addr_mcast_scope(group, 0x02))) {
+		return NULL;
+	}
```

Comment:
empty line after }

Reply:
Done. Not hit by automated check cause after the marked line.